modulefiles/wcoss2/fv3

```diff
@@ -1,53 +1,95 @@
-#%Module######################################################################
-##
-##    NEMS FV3 Prerequisites: Cactus and Dogwood (WCOSS2)
-
-proc ModulesHelp {} {
-  puts stderr "\tcit - loads modules required for building and running FV3 under NEMS on WCOSS2"
-}
-
-module-whatis "loads NEMS FV3 prerequisites on Cactus and Dogwood"
-
-# NOTE: the "module purge" and loading of the module command are
-# handled by the module-setup.sh (or .csh) script.
-
-##
-## load programming environment
-## this typically includes compiler, MPI and job scheduler
-##
-module load PrgEnv-intel/8.1.0
-module load intel/19.1.3.304
-module load craype/2.7.10
-module load cray-mpich/8.1.9
-
-module load cmake/3.20.2
-
-# other nceplibs
-module load bacio/2.4.1
-module load ip/3.3.3
-module load sp/2.3.3
-module load w3nco/2.4.1
-module load w3emc/2.9.2
-module load nemsio/2.5.2
-module load g2tmpl/1.9.1
-module load crtm/2.3.0
-module load g2/3.4.5
-module load upp/8.1.0
-
-# WW3 grib encoding
-module load jasper/2.0.25
-module load libpng/1.6.37
-module load zlib/1.2.11
-
-# hdf5, netcdf, esmf
-module load hdf5/1.10.6
-module load netcdf/4.7.4
-module load esmf/8.0.1
-
-##
-## load cmake
-##
-setenv CMAKE_C_COMPILER cc
-setenv CMAKE_CXX_COMPILER CC
-setenv CMAKE_Fortran_COMPILER ftn
-setenv CMAKE_Platform wcoss2
+help([[
+loads NEMS FV3 prerequisites on Cactus and Dogwood
+]])
+
+PrgEnv_intel_ver=os.getenv("PrgEnv_intel_ver") or "8.1.0"
+load(pathJoin("PrgEnv-intel", PrgEnv_intel_ver))
+--module load PrgEnv-intel/8.1.0
```

Comment:
I would remove these commented 'module load ..' commands.

Reply:
all the module version comments are removed